x-pack/test/functional/services/uptime/alerts.ts

```diff
@@ -104,11 +104,11 @@ export function UptimeAlertsProvider({ getService }: FtrProviderContext) {
       await testSubjects.click('uptimeAlertAddFilter.monitor.type');
       await testSubjects.click('uptimeCreateStatusAlert.filter_scheme');
     },
-    async clickSaveAlertButton() {
-      await testSubjects.click('saveAlertButton');
+    async clicksaveRuleButton() {
```

Comment:
Why did the casing of this function change?

Reply:
I know why because I did bad `CTRL-H`, I can put it back